### DISCUSSION TITLE
Latest Pytorch

### DISCUSSION THREAD

**Discussion by meghbhalerao** [Lightning-AI/pytorch-lightning#16046] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Hi everyone, this is a very basic question but is pytorch lightning always updated with the latest pytorch release?
Thanks!
Megh

**Comment by akihironitta** [comment]
Yes! See also https://pytorch-lightning.readthedocs.io/en/latest/versioning.html#pytorch-support
